Neos.ContentRepository.Core/Classes/Feature/NodeTypeChange/NodeTypeChange.php

```diff
@@ -420,11 +426,13 @@ private function findDimensionSpacePointsConnectingParentAndChildAggregate(
     }
 
     private function removeNodeInDimensionSpacePointSet(
+        ContentGraphInterface $contentGraph,
         NodeAggregate $nodeAggregate,
         DimensionSpacePointSet $coveredDimensionSpacePointsToBeRemoved,
     ): NodeAggregateWasRemoved {
         return new NodeAggregateWasRemoved(
-            $nodeAggregate->contentStreamId,
+            $contentGraph->getWorkspaceName(),
+            $contentGraph->getContentStreamId(),
```

Comment:
Previously the `contentStreamId` was taken from the node aggregate – but since that does not contain the `workspaceName` I changed both to use the info from the `ContentGraph`.
Not 100% sure about this, but in practice it should not make a difference

Reply:
actually that seems like the best option as `NodeAggregate::contentStreamId` will be removed.